### PR TITLE
Add infobox to inject scrobbling info & state into pages

### PR DESCRIPTION
**Pull request by Boelensman1**
**Describe the changes you made**
Add infobox to display scrobbling state on the page. My motivation is that I dislike notifications, it would also be a good solution for safari where you're apparently not allowed to send notifications.

**Additional context**
image

**Todo**
- [x] Allow connectors to set custom styling
- [x] Make optional
- [x] Use translations for the scrobbling/loading etc. text
- [x] Improve documentation of the code
- [ ] More thorough testing

### CODE REVIEW

**Comment by Boelensman1** [comment]
Oops, apparently when you reload the page github changes draft pull request back into a normal one. And now I can't seem to change it into one. So just to be clear, this was supposed to be a draft...

**Comment by Boelensman1** [comment]
All right, this should be about done.

I did have the issue that the extension only started doing things after I clicked on it, but I don't think that's related to my code and might just be a bug in the dev environment?

**Comment by yayuyokitano** [comment]
I have created a PR with suggested change: https://github.com/Boelensman1/web-scrobbler/pull/1

**Review by yayuyokitano — CHANGES_REQUESTED** [review]
In addition to the mentioned PR

File: src/_locales/en_GB/messages.json [hunk 1]
```diff
@@ -774,5 +774,17 @@
     "extDescription": {
         "message": "Scrobble music all around the web!",
         "description": "Description of the extension"
+    },
+    "infoBoxStateDisallowed": {
+        "message":  "Scrobbling is not allowed due to an option",
```
[the remark this answers is not in the document]
Comment (Boelensman1) [reply src/_locales/en_GB/messages.json:779]: Ah, I basically copied the comment from https://github.com/web-scrobbler/web-scrobbler/blob/7f598dc6922a25978162e6730455181ab915d97f/src/core/object/controller/controller-mode.ts#L6-L9 Do you have a suggestion for a better description of what disallowed means?

[the remark this answers is not in the document]
Comment (yayuyokitano) [reply src/_locales/en_GB/messages.json:779]: I dont know how to do so concisely. I think we can do one of those where we tell the user to open the popup maybe?

[the remark this answers is not in the document]
Comment (Boelensman1) [reply src/_locales/en_GB/messages.json:779]: Sounds like a good idea. I tried for a bit to make the icon clickable and open the popup, which is possible but not from the content script. Would be cool for a future update.

[the remark this answers is not in the document]
Comment (yayuyokitano) [reply src/_locales/en_GB/messages.json]: Seems trivial. I'll handle it once this is merged. I'll give this another review tomorrow and try to have a look into the test issue.

**Comment by yayuyokitano** [comment]
https://github.com/Boelensman1/web-scrobbler/pull/2

**Review by yayuyokitano — APPROVED** [review]
I feel this is good to go now. Thank you for all the work!

**Pull request merged**